_includes/snippets/saml/auth/response_example.md

```diff
@@ -1,38 +1,50 @@
 {% capture example %}
 ```xml
-<samlp:Response ID="_b28d50c0-dc35-0134-96f3-06d8bac14e9d"
-                Version="2.0"
-                IssueInstant="2017-02-23T20:36:37Z"
-                Destination="https://sp.int.identitysandbox.gov/auth/saml/callback"
-                Consent="urn:oasis:names:tc:SAML:2.0:consent:unspecified"
-                InResponseTo="_6fca7b78-9ab7-49f5-bd62-18c48eac3c68"
-                xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol">
+<samlp:Response xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" ID="_7f3d8cd9-d3f8-4b47-a571-5272810d5073" Version="2.0" IssueInstant="2024-09-18T16:20:36Z" Destination="https://sp.int.identitysandbox.gov/auth/saml/callback" Consent="urn:oasis:names:tc:SAML:2.0:consent:unspecified" InResponseTo="_bf054c05-5b2c-4773-a6a9-9ba075a87bc9">
```

Comment:
[Question] Are the Response attributes intentionally placed on a single line? Is there a benefit over the existing layout?

Reply:
It wasn't intentional. It is what [SAML Tool's](https://www.samltool.com/) XML formatter gave me. It does substitute vertical scrolling (which I prefer) for horizontal scrolling. Do we have a preference?

Reply:
Good question, I only noticed that it was a change. I don't think either way is particularly readable.

Reply:
In my opinion, the horizontal scrolling is better as it allows you to understand the structure from the "pretty-printed" indentation. Would anyone else like to weigh in with their opinions?

_Note: Existing content in https://developers.login.gov/saml/authentication/#example-specifying-ial-aal-and-attributes follows horizontal scrolling._